app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/health/AppTPCPUMonitor.kt

```diff
@@ -35,6 +37,9 @@ import javax.inject.Inject
 class AppTPCPUMonitor @Inject constructor(
     private val workManager: WorkManager,
     private val appTpFeatureConfig: AppTpFeatureConfig,
+    private val appBuildConfig: AppBuildConfig,
+    private val context: Context,
```

Comment:
I would remove the `context` parameter from here. It can be directly injected in the `CPUPerformanceLogger`